bindings/matrix-sdk-ffi/src/authentication_service.rs

```diff
@@ -40,7 +40,7 @@ pub struct AuthenticationService {
     base_path: String,
     passphrase: Option<String>,
     user_agent: Option<String>,
-    client: RwLock<Option<Client>>,
+    client: tokio::sync::RwLock<Option<Client>>,
```

Comment:
instead, can we use import renamings please (and remove all qualified paths with `tokio::sync` introduced in this patch)? `StdRwLock` for the libstd one, `AsyncRwLock` for the tokio one.